libmozdata/patchanalysis.py

```diff
@@ -750,6 +750,15 @@ def _parse_line(h, v):
         if v != '':
             out[h].append(v)
 
+    # Remove html entities
+    html_escape_table = {
+        '&': '&amp;',
+        ''': '&quot;',
+        ''': '&apos;',
+        '>': '&gt;',
+        '<': '&lt;',
+    }
+    text = ''.join(html_escape_table.get(c, c) for c in text)
```

Comment:
Nit: You could use `escape` from `xml.sax.saxutils` to make it faster, but it doesn't really matter.